modles/getModel.js

```diff
@@ -50,10 +50,22 @@ function getarticle_cmntMdl(req) {
       else return result.rows;
     });
 }
-
+function getUsersMdl(req) {
+  return db
+    .query("SELECT * FROM users")
+    .then((result) => {
+      if (result.rowCount === 0)
+        return Promise.reject({ status: 404, msg: "not found" });
+      else return result.rows;
+    })
+    .catch((err) => {
```

Comment:
we don't need a `.catch` in the model if we have one in the controller, as if this promise rejects with an error it will send it back to the controller which will get it in the .catch block